runtime/lua/vim/diagnostic.lua

```diff
@@ -338,6 +338,32 @@ local function make_augroup_key(namespace, bufnr)
   return string.format('DiagnosticInsertLeave:%s:%s', bufnr, ns.name)
 end
 
+---@private
+local function execute_scheduled_display(namespace, bufnr)
+  local args = bufs_waiting_to_update[bufnr][namespace]
+  if not args then
+    return
+  end
+
+  -- Clear the args so we don't display unnecessarily.
+  bufs_waiting_to_update[bufnr][namespace] = nil
+
+  M.show(namespace, bufnr, nil, args)
+end
+
+--- @deprecated
+--- Callback scheduled when leaving Insert mode.
+---
+--- called from the Vimscript autocommand.
+---
+--- See @ref schedule_display()
+---
+---@private
+function M._execute_scheduled_display(namespace, bufnr)
+  vim.deprecate('vim.diagnostic._execute_scheduled_display', 'execute_scheduled_display', '0.9')
```

Comment:
The second argument to `vim.deprecate` is "what to use instead". In this case, there is nothing to use instead, so leave it `nil`.

```suggestion
  vim.deprecate('vim.diagnostic._execute_scheduled_display', nil, '0.9')
```